app/database.py

```diff
@@ -0,0 +1,30 @@
+from app import app
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField
+from wtforms.validators import DataRequired
+import os
+
+app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("SQLALCHEMY_DATABASE_URI")
+db = SQLAlchemy()
+db.init_app(app)
```

Comment:
Db is rather initialized in the same file where is run 
https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xv-a-better-application-structure

When you will have more models it will be obvious where to import db from